scvi/model/_jaxscvi.py

```diff
@@ -132,6 +132,7 @@ def module(self):
     def train(
         self,
         max_epochs: Optional[int] = None,
+        check_val_every_n_epoch: Optional[int] = None,
```

Comment:
should we always append new kwargs to the end of the signature unless it really doesnt make sense? Just to avoid situations when they are used as args.

Reply:
yes, but this whole API is experimental. Right now for scvi et al. this is part of **kwargs to the trainer. I think this is fine for now.